app/controllers/api/v1/hotels_controller.rb

```diff
@@ -0,0 +1,85 @@
+class Api::V1::HotelsController < ApplicationController
+  # before_action :logged_in, except: %i[index show]
+
+  # before_action :user_ability, except: %i[index show]
```

Comment:
- [ ] Please, remove these old pieces of commented code to improve the cleanliness of the repo 💪

Reply:
Resolved, thanks.